src/osd/modules/netdev/vmnet_common.cpp

```diff
@@ -0,0 +1,234 @@
+// license:BSD-3-Clause
+// copyright-holders: Kelvin Sherlock, R. Belmont
+
+#include "emu.h"
+#include "vmnet_common.h"
```

Comment:
You shouldn’t `#include "emu.h"` here – we’re trying to reduce circular dependencies.  I realise it’s still needed in netdev modules, but it really shouldn’t be needed in a utility file like this.